src/shogun/classifier/LDA.h

```diff
@@ -173,7 +173,7 @@ class CLDA : public CLinearMachine
 		/** @return object name */
 		virtual const char* get_name() const { return "LDA"; }
 
-	protected:
+	public:
```

Comment:
working with friend classes is infeasible here?

Reply:
i have not tried yet

Reply:
did you try the friend class thingi? would be nicer to  not have this method public. minor though